api/pom.xml

```diff
@@ -90,12 +94,126 @@
         </site>
     </distributionManagement>
 
-    <modules>
-        <module>activation</module>
-    </modules>
-
     <build>
-        <defaultGoal>install</defaultGoal>
+
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
+                <excludes>
+                    <exclude>META-INF/*.default</exclude>
+                </excludes>
+            </resource>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>false</filtering>
+                <includes>
+                    <include>META-INF/*.default</include>
+                </includes>
+            </resource>
+        </resources>
+
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.glassfish.hk2</groupId>
+                    <artifactId>osgiversion-maven-plugin</artifactId>
+                    <version>3.0.2</version>
```

Comment:
this plugin can go away, it is no longer used by the build

Reply:
Done.